pkg/tarsplitutils/tarstream.go

```diff
@@ -0,0 +1,138 @@
+package tarsplitutils
+
+import (
+	"io"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/vbatts/tar-split/tar/storage"
+)
+
+func min(x, y int64) int64 {
```

Comment:
No reason other than that's the types I have in the places I use the respective functions